### PR TITLE
Programmatically Configure SonarQube settings in Jenkins Ephemeral

### PR DESCRIPTION
**Pull request by InfoSec812**
#### What does this PR do?
Handles creating and configuring an auth token for use with SonarQube inside of Jenkins. 

#### Is there a relevant Issue open for this?
N/A

#### Are there any other relevant resources that should be reviewed as well?
N/A

#### Who would you like to review this?
/cc @sherl0cks

### CODE REVIEW

**Comment by InfoSec812** [comment]
One thing to keep in mind... If Jenkins gets built and starts up before SonarQube is ready, a redeploy will have to be initiated once SonarQube IS ready.

File: configuration/init.groovy.d/configure-jenkins.groovy [hunk 1]
```diff
@@ -15,6 +31,58 @@ if ( Jenkins.instance.views.findAll{ view -> view instanceof com.smartcodeltd.je
   Jenkins.instance.addView(view)
 }
 
+LOG.log(Level.INFO, 'Get SonarQube config')
```
Comment (sherl0cks) [inline configuration/init.groovy.d/configure-jenkins.groovy]: let's wrap this operation in an environment variable switch as discussed today

File: plugins.txt [hunk 2]
```diff
@@ -1,98 +1,147 @@
 ace-editor:1.1
+analysis-core:1.92
```
Comment (sherl0cks) [inline plugins.txt]: the diff on this file confuses me. I diffed the files locally and saw no changes.

Comment (InfoSec812) [reply plugins.txt]: Not sure why it's different other than I think I cherry-picked this bit from another branch.

**Pull request merged**